### PR TITLE
Changed spacing for upcoming events

### PR DESCRIPTION
**Pull request by Mike865**
Changed grid component sizes so that the mobile page will not over flow and look better. Changed the weekday, date day, and date year attribute sizes to be bigger. Changed outline of date information from gray to blue. Removed icon from see details button and changed icon for the add to calendar button. Added a color in theme.js called outline for the outline color of the date box.

### CODE REVIEW

File: src/components/EventCardFlip.js [hunk 1]
```diff
@@ -1,144 +1,148 @@
...
-import {
-  Stack,
-  Grid,
-  Typography,
-  Button,
-  Paper,
-} from "@mui/material";
+import AddIcon from '@mui/icons-material/Add';
+import {useTheme} from "@mui/material/styles";
+import { Stack, Grid, Typography, Button, Paper } from "@mui/material";
```
Comment (Sanjay-B) [inline src/components/EventCardFlip.js:6]: Overall text might be too big. Defer to @aimiisme for design review. cc: @aimiisme @PremKSU

File: src/components/EventCardFlip.js [hunk 2]
```diff
@@ -1,144 +1,148 @@
...
+                  component="div"
+                  sx={{ fontSize: { xs: 22, sm: 26, md: 30, lg: 30 } }}
                 >
-                    {props.title}
+                  {props.title}
                 </Typography>
                 <Typography variant="body1" gutterBottom>
-                    Location: <a href={props.locationData}>{props.locationData}</a>
+                  Location:{" "}
+                  <a href={props.locationData}>{props.locationData}</a>
```
Comment (Sanjay-B) [inline src/components/EventCardFlip.js:69]: Does not wrap correctly in mobile view. Refer to picture. 

Screen Shot 2022-11-10 at 12 41 02 PM

cc: @PremKSU

**Review by Sanjay-B — CHANGES_REQUESTED** [review]
Few feedback comments. Programmatically, this looks good. Design-wise, up to @aimiisme.

File: src/components/EventCardFlip.js (hunk 2 above)
Comment (aimiisme) [reply src/components/EventCardFlip.js:69]: I wiil work with @Mike865 to resolve this issue soon.

File: src/components/EventCardFlip.js (hunk 1 above)
Comment (aimiisme) [reply src/components/EventCardFlip.js:6]: @Mike865 has reduced the text size in his latest commit and they look good to me.

**Review by aimiisme — COMMENTED** [review]
@Sanjay-B I'm experiencing some difficulty viewing the event flip card on my computer (I believe it's just not working for me.) However, I have reviewed the screenshot that Michael sent me, and the overall design looks good to me. Would you mind checking if the hyperlink works for each `location.type` that we discussed?

**Comment by Sanjay-B** [comment]
> @Sanjay-B I'm experiencing some difficulty viewing the event flip card on my computer (I believe it's just not working for me.) However, I have reviewed the screenshot that Michael sent me, and the overall design looks good to me. Would you mind checking if the hyperlink works for each `location.type` that we discussed?

image

I rebalanced this branch with ``origin/dev``. It produces the error above. Do you see the same? @aimiisme , @Mike865

**Review by Sanjay-B — COMMENTED** [review]
Refer to earlier comment. This looks good but, ``http://localhost:3000/events`` does not load for me.

**Comment by Sanjay-B** [comment]
Update: Since ``/events`` is not part of RC0, I am going to defer this to RC1. @Mike865 cc: @aimiisme

**Review by Sanjay-B — APPROVED** [review]
LGTM.

**Comment by Sanjay-B** [comment]
I am closing this because it's stale. Feel free to reopen if need be. @Mike865 cc: @aimiisme , @Hoangnguy02 , @zainabm9786

**Pull request closed without merging**